src/Microsoft.Framework.Runtime/NuGet/Utility/VersionUtility.cs

```diff
@@ -251,7 +246,7 @@ public static FrameworkName ParseFrameworkName(string frameworkName)
                     return UnsupportedFrameworkName;
                 }
 
-                version = _emptyVersion;
+                version = identifierPart.Equals(NetPlatformFrameworkIdentifier) ? new Version(5, 0) : _emptyVersion;
```

Comment:
This is turning out to be a bit weird. So I'm going to drop it. The Version Number for "dotnet" will be "0.0". We can display it as "5.0" in the places we want to, if we want to.